.readthedocs.yml

```diff
@@ -3,5 +3,5 @@ build:
 python:
   version: 3.6
   extra_requirements:
-  - docs
+    - docs
```

Comment:
I'd like to avoid indenting this.

Reply:
...why? It is the default in yamllint and more readable too (since the list is a child item it should be indented). Also the yaml files used by molecule are styled like this.

Reply:
I'd argue that too much indentation creates more distraction and more ugliness. Also, since the files changed are mostly maintained by me I'll be insisting.
You'll see when you end up with 10 levels of indentation someday.

It's fine to lint that it's parsable, though.

Reply:
So we need a "files that webknjaz maintains" exclude for yamllint? This is the second person who wants to indent this vs. your "I maintain (own!?) this file" mentality. Don't stop imparting your wisdom on this project, please, but your subjectivities around indentation may be having less interesting effects. I only speak for myself ...

Reply:
Well, you were caring about contributor experience. So from this PoV it's not logical to make things unreadable for the contributor who looks there often vs. the possibility of somebody else looking at it once or twice in the next decade. I wouldn't care if I were to fully defer the related responsibilities to someone else.

> exclude for yamllint

a separate config would do

P.S. In the internal Ansible Core team Slack it looks like the majority of folks prefer non-indented sequences (as per recent discussions). We also mostly disagree with a lot of what ansible-lint pushes by default. I think this may lead to PoV diversity.

Reply:
I see a number of comments telling me about experience now. Please do not use this as a bat to beat me with. My personal experience is fine except for trying to guess at what others might find easier and then hearing the fall out from it ... but I am going to stop doing that now.

Reply:
The travis file is also heavily (and unnecessarily in my opinion) parametrized with tons of single line anchors etc.

I wouldn't see the indentation of the file as the main source of its distractiveness and ugliness.

> P.S. In the internal Ansible Core team Slack it looks like the majority of folks prefer non-indented sequences (as per recent discussions). We also mostly disagree with a lot of what ansible-lint pushes by default. I think this may lead to PoV diversity.

Then the internal slack group is free to comment here as well and give their reasons. I already wrote mine above (having the `-`on the same indentation level while being a child item makes it less obvious that this is a child element). Also this is about yamllint, not ansible-lint.

As a data point: https://yaml.org/spec/1.2/2009-07-21/spec.html#id2559116 indents child lists.